src/frontend/src/apis/Users.api.ts

```diff
@@ -35,11 +35,19 @@ export const getSingleUser = (id: number) => {
  * @param id_token The login token for the user.
  */
 export const logUserIn = (id_token: string) => {
-  return axios.post<AuthenticatedUser>(
-    apiUrls.usersLogin(),
-    { id_token },
-    { transformResponse: (data) => authUserTransformer(JSON.parse(data)) }
-  );
+  return axios
+    .post<AuthenticatedUser>(
+      apiUrls.usersLogin(),
+      { id_token },
+      { transformResponse: (data) => authUserTransformer(JSON.parse(data)) }
+    )
+    .then((res) => {
+      const token = res.headers['authorization'];
+      localStorage.setItem('token', token);
```

Comment:
I definitely don't think we should be storing tokens in local storage since that's the most easily accessed option

Reply:
meh, tried to do httponly cookies but it doesn't work, get crazy cors errors